templates/quickstart-confluence-master.template.yaml

```diff
@@ -1163,10 +1163,10 @@ Resources:
           PropagateAtLaunch: true
         # NOTE: The leading COMMIT/TIMESTAMP are used to locate the position to update; see scripts/update-tags.py
         - Key: "atl:quickstart:commit-id"
-          Value: "COMMIT: fc08aac3b7c33646e6db6085460fc7cd3506d8e8"
+          Value: "COMMIT: 0b0d3d379491d4990df070feb6a7b498485321da"
```

Comment:
I am not sure if we should change the commits for these changes. They are just test related so I would keep them same.